src/main/java/frc/trigon/robot/constants/OperatorConstants.java

```diff
@@ -23,11 +23,10 @@ public class OperatorConstants {
             RESET_HEADING_TRIGGER = DRIVER_CONTROLLER.y(),
             SET_GYRO_HEADING_TO_SOLVE_PNP_HEADING_TRIGGER = OPERATOR_CONTROLLER.r().or(DRIVER_CONTROLLER.a()),
             TOGGLE_BRAKE_TRIGGER = OPERATOR_CONTROLLER.g().or(RobotController::getUserButton),
-            TOGGLE_FIELD_AND_SELF_RELATIVE_DRIVE_TRIGGER = DRIVER_CONTROLLER.b(),
+            TOGGLE_JOYSTICK_ORIENTED_ROTATION_TRIGGER = DRIVER_CONTROLLER.b(),
```

Comment:
Say both kinds of rotation in the name

Reply:
again, I think this clutters it unnecacerily